src/handlers/forex.js

```diff
@@ -0,0 +1,30 @@
+const casaLib = require('@mojaloop/finance-portal-lib');
+const util = require('util');
+
+const { getFxpRatesPerCurrencyChannel } = casaLib.admin.api;
+
+const handler = (router, routesContext) => {
+    router.get('/forex/rates', async (ctx, next) => {
+        let fxpRatesPerCurrencyChannel;
+
+        try {
+            fxpRatesPerCurrencyChannel = await getFxpRatesPerCurrencyChannel(
```

Comment:
Couldn't you do the ctx.response.body, status, and await next() here so this could be const?